engine/gamesys/src/gamesys/components/comp_collection_factory.cpp

```diff
@@ -63,34 +81,36 @@ namespace dmGameSystem
     dmGameObject::CreateResult CompCollectionFactoryNewWorld(const dmGameObject::ComponentNewWorldParams& params)
     {
         CollectionFactoryContext* context = (CollectionFactoryContext*)params.m_Context;
-        FactoryWorld* fw = new FactoryWorld();
+        CollectionFactoryWorld* world = new CollectionFactoryWorld();
```

Comment:
Keeping the "world" and "component" naming scheme consistent throughout the engine.